apps/indexer-coordinator/src/utils/contractSDK.ts

```diff
@@ -43,6 +43,7 @@ export const networkToChainID: Record<SubqueryNetwork, ChainID> = {
 
 export const sdkOptions = {
   [ChainID.testnet]: createContractOptions('testnet'),
+  [ChainID['testnet-mumbai']]: createContractOptions('testnet-mumbai'),
```

Comment:
Can remove `testnet-mumbai`

Reply:
type SubqueryNetwork = "local" | "testnet" | "testnet-mumbai" | "mainnet"
type from contract sdk has to be updated first